apis/v1alpha1/shared_types.go

```diff
@@ -175,6 +189,33 @@ type AdminNetworkPolicyEgressPeer struct {
 	// <network-policy-api:experimental>
 	// +optional
 	Nodes *metav1.LabelSelector `json:"nodes,omitempty"`
+	// Networks defines a way to select peers via CIDR blocks (both v4 & v6).
+	// This is intended for representing entities that live outside the cluster,
+	// which can't be selected by pods, namespaces and nodes peers, but note
+	// that cluster-internal traffic will be checked against the rule as
+	// well. So if you Allow or Deny traffic to `"0.0.0.0/0"`, that will allow
+	// or deny all IPv4 pod-to-pod traffic as well. If you don't want that,
+	// add a rule that Passes all pod traffic before the Networks rule.
+	//
+	// Do not provide serviceCIDR range in `networks` peer. This is NOT supported.
+	// While 0.0.0.0/0 will match on the serviceVIP as well, it does not make much sense
+	// anyways in that case because policies are expected to be applied after the serviceVIP
+	// (clusterIP, externalIP, loadbalancer.Ingress.VIP) is rewritten to the actual backend IPs.
+	// So policy match finally gets applied on the endpointIPs not serviceVIPs.
+	//
+	// Each item in Networks should be provided in the CIDR format and should be
+	// IPv4 or IPv6, for example "10.0.0.0/8" or "fd00::/8". IPv4 address embedded
+	// in IPv6 addresses are not supported.
```

Comment:
```suggestion
	// IPv4 or IPv6, for example "10.0.0.0/8" or "fd00::/8".
```

ipv4-embedded addresses are not something that most people ever need to think about, so mentioning them at all is probably going to confuse more people than it helps.

Reply:
ack removed.